Sources/Classes/RSClient.m

```diff
@@ -240,6 +240,9 @@ - (void) alias:(NSString *)newId options:(RSOption *) options {
     if(prevId == nil) {
         prevId =[traits objectForKey:@"id"];
     }
+    if (prevId == nil) {
+        prevId = self.anonymousId;
+    }
```

Comment:
To avoid multiple if cases, use Ternary Conditional Operator(?:) like below.

`NSObject *prevId = [traits objectForKey:@"userId"] ?: [traits objectForKey:@"id"] ?: self.anonymousId;`